servers/trunk/resources/log4j.properties

```diff
@@ -19,6 +19,6 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
 
-log4j.logger.kafka=DEBUG, stdout
+log4j.logger.kafka=INFO, stdout
```

Comment:
Why?

Reply:
My bad. This was just for some tests and slipped into the commit. I'll revert this.